test/integration/cli-args/filename_error.expected

```diff
@@ -0,0 +1,13 @@
+  $ ../../../../install/default/bin/stanc --filename-in-msg=foo.stan  filename_error.stan
+
+Semantic error in 'filename_error.stan', line 3, column 17 to column 18:
```

Comment:
This should be " Semantic error in 'foo.stan' "

Reply:
woops

Reply:
The stancjs tests pass so I think this is something minor.